tests/annotation_merge.nf.test.snap

```diff
@@ -110,6 +110,8 @@
             [
                 "annotation",
                 "annotation/test",
+                "annotation/test/test_snpEff.ann.vcf.gz",
+                "annotation/test/test_snpEff.ann.vcf.gz.tbi",
```

Comment:
yeah, that's why we had the saveAs closure in conf/modules/annotate.config

Reply:
Ah okay, thanks I'll revert that change